### PR TITLE
Fix Who's That Pokemon for Webclient

### PR DESCRIPTION
**Pull request by ET54321**

### CODE REVIEW

File: scripts/trivia.js [hunk 1]
```diff
@@ -737,6 +737,16 @@ TriviaGame.prototype.startTriviaRound = function () {
...
                 i++;
             }
+            // to account for webclient until it gets updated to Gen 7
+            if ((triviaq.get(questionNumber) === null || (triviaq.get(questionNumber).category.toLowerCase() === "who's that pokémon?" && this.webclientPlayers())) && i !== 200) {
+                var qu = triviaq.get(questionNumber);
+                if (qu.question > 721) { // the stored questions for who's that pokemon? are the dex numbers. Basically, "if this is > Gen 7"
+                    var ran = sys.rand(1, 721);
+                    // all the who's that pokemon questions are stored sequentially because they were created with /createwtp
+                    // so let's change to a pre-Gen 7 pokemon
+                    questionNumber = questionNumber - ran;
```
Comment (PokeWorldBW) [inline scripts/trivia.js]: Er, how exactly does this work? Let's say the Pokemon is Torracat and ran is 1, won't that change the question to Litten?

**Comment by ET54321** [comment]
I was trying to get it to pick a different Who's That Pokemon? question if it picked a Gen 7 Pokemon with webclient players in the game; however, that was a big flaw in my logic. For the next commit, I just opted to get it to pick a different question, whether it is WTP or not, until it picks a question that will work.

I also added the ability to adjust some of the question timers to make debugging easier, but that's a separate topic.

**Comment by PokeWorldBW** [comment]
I mean, you still could have done that by doing questionNumber -= 81 which would be enough to guarantee that the new Pokemon wouldn't be from Gen 7. It would cause the same Pokemon to correspond to each Gen 7 Pokemon though. If you wanted to make it random, you could make it like questionNumber = questionNumber - 801 + sys.rand(1, 722)
But it looks like you opted to go for another option, which I guess works and does something different.

**Comment by StrudelsPO** [comment]
am i missing something here or can you not just do questionNumber = sys.rand(1, 722);?

**Comment by ET54321** [comment]
Yea, I thought about just doing -100 or something, but I didn't want it to correspond to the same thing. The latter solution wouldn't work either because if it was something like Rowlett with a low random number, then it would be like -801 +1, and you would step into a different category since WTP only has 801 questions. Something like: 

=questionNumber - 80 - sys.rand(1,( 722 - 80)) 

where 80 is the number of Gen 7 Pokemon + 1, should work because the first 80 ensures that you get out of Gen 7 even if the dex number is 801, and the next 80 ensures that you don't step out of the category if it is Rowlett.
Meanwhile, this is all for a temporary fix until the Webclient is compatible with the Gen 7 Pokemon, so it is probably better that I went with simply "ask a different question". 

@StrudelsPO questionNumber is the question ID in the question database (so think something like 9820), rather than the Pokedex number, but all of the questions for the category are stored in an 801 unit block.

Bottom line, the current commit avoids all of this and simply gets a different question.

**Comment by StrudelsPO** [comment]
o
